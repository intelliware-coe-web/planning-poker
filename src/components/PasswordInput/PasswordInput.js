import React from 'react';

function PasswordInput(props) {
    return (
        <div className="uk-margin">
            <div className="uk-inline">
                <span className="uk-form-icon" uk-icon="icon: lock"></span>
                <input className="uk-input" type="password" placeholder="Password" />
            </div>
        </div>
    );
}

export default PasswordInput;