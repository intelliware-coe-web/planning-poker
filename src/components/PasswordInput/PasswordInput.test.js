import React from 'react';
import PasswordInput from './PasswordInput';

it('password input should match snapshot', () => {
    expect(PasswordInput()).toMatchSnapshot();
});