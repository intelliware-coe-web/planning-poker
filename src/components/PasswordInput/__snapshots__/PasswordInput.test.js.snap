// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`password input should match snapshot 1`] = `
<div
  className="uk-margin"
>
  <div
    className="uk-inline"
  >
    <span
      className="uk-form-icon"
      uk-icon="icon: lock"
    />
    <input
      className="uk-input"
      placeholder="Password"
      type="password"
    />
  </div>
</div>
`;
